armi/utils/asciimaps.py

```diff
@@ -281,8 +493,8 @@ def asciiMapFromGeomAndSym(geomType: str, symmetry: str):
     symmetry = symmetry.replace(geometry.THROUGH_CENTER_ASSEMBLY, "")
 
     MAP_FROM_GEOM = {
-        (geometry.HEX, geometry.THIRD_CORE): AsciiMapHexThird,
-        (geometry.HEX, geometry.FULL_CORE): AsciiMapHexFullTipsUp,
+        (geometry.HEX, geometry.THIRD_CORE): AsciiMapHexThirdFlatsUp,
+        (geometry.HEX, geometry.FULL_CORE): AsciiMapHexFullFlatsUp,
```

Comment:
This is causing the build failure. Will need to have more dimensions of choice here because we now need to specify Flats up or Tips up depending on if it's a core or a pin array in an assembly.